src/draw/DefaultDrawer.js

```diff
@@ -350,13 +350,18 @@ class DefaultDrawer {
     const origParent = this.pluginData.getComponentById(event.subject.parent.data.id);
     const target = dropTarget ? d3.select(dropTarget) : null;
 
-    if (target === origParent) {
-      const { x, y } = event.subject;
+    if (target === origParent
+      || (target && target.datum().data && origParent
+        && origParent.id === target.datum().data.id)) {
```

Comment:
bad indentation:

expect to have :
```js
    if (target === origParent
      || (target && target.datum().data
      && origParent
      && origParent.id === target.datum().data.id)) {
```

can we do this 
`origParent?.id === target.datum().data.id`
instead of 
`&& origParent && origParent.id === target.datum().data.id)`
 ?

Reply:
updated